docs/installation/index.md

```diff
@@ -0,0 +1,344 @@
+	Use the <span class="navigation-hint_desktop">**navigation tree on the left**</span><span class="navigation-hint_mobile">**main menu button in the upper-left corner**</span> to navigate through the documentation. Newly added or meaningfully updated guides are marked with a <span class="tree__item__badge tree__item__badge_new">NEW</span> icon for easy spotting.
+</info-box>
+
+<info-box>
+	**Not sure which installation method is best for you?** <button type="button" class="quiz-button quiz-button_start">Take a short quiz!</button>
+</info-box>
+
+## Installing CKEditor 5
+
+Learn how to install CKEditor 5 easily with the use of {@link installation/getting-started/quick-start#creating-custom-builds-with-online-builder online builder} or {@link installation/getting-started/quick-start#building-the-editor-from-source build the editor from scratch}, learn to {@link installation/getting-started/quick-start#building-the-editor-from-source create a custom build}, {@link installation/getting-started/configuration configure it}, and {@link installation/advanced/csp secure}.
+
+## Predefined CKEditor 5 builds
+
+Predefined CKEditor 5 builds are ready-to-use installations aimed at specific needs that you can simply download and use out of the box. Learn more about the {@link installation/advanced/predefined-builds available predefined builds} and choosing the right one for you. This is the fastest way to kick-off you CKEditor 5 installation.
+
+## Integration with frameworks
+
+Get to know the supported {@link installation/frameworks/overview integrations with popular JavaScript frameworks} such as React, Angular or Vue, and learn to utilize them and to integrate CKEditor 5 with you software.
+
+## Migrating from CKEditor 4
+
+Refer to the {@link installation/getting-started/migration-from-ckeditor-4 CKEditor 4 migration guide} if you are updating your CKEditor 5 installation.
+
+## Advanced installation concepts
+
+Find out more about the {@link installation/advanced/plugins plugin development}, how to {@link installation/advanced/saving-data handle and save the data} and what the {@link installation/advanced/features-html-output-overview features' HTML output} is for each plugin. Learn about alternative setups such as {@link installation/advanced/dll-builds DLL builds} or {@link installation/advanced/integrating-from-source integrating CKEditor 5 from source}.
+
+## Related links
+
+ * {@link updating/index Updating CKEditor 5} &ndash; Find out how to keep you installation up-to-date at all times.
+ * {@link features/index Features} &ndash; Learn more about the CKEditor 5 features.
+ * {@link examples/index Examples} &ndash; Try live demos of available predefined builds and custom solutions.
+ * * {@link framework/guides/overview CKEditor 5 Framework} &ndash; Learn how to work with CKEditor 5 Framework, customize it, create your own plugins or custom editors, how to change the UI or even bring your own UI to the editor.
+
+<script type="text/javascript">
```

Comment:
Couldn't we extract the `<script>` source to a new snippet? Then, in a snippet HTML file, execute it.

```html
<script>startQuiz();</script>
```

Reply:
It will be covered in a follow-up: https://github.com/cksource/ckeditor5-internal/issues/1577.